Simplenote/src/main/java/com/automattic/simplenote/authentication/passkey/PasskeyManager.kt

```diff
@@ -0,0 +1,38 @@
+package com.automattic.simplenote.authentication.passkey
+
+import android.content.Context
+import android.util.Log
+import androidx.credentials.CreatePublicKeyCredentialRequest
+import androidx.credentials.CredentialManager
+import androidx.credentials.exceptions.CreateCredentialException
+import com.automattic.simplenote.Simplenote
+import com.automattic.simplenote.viewmodels.PasskeyViewModel
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+
+const val TAG = "PasskeyManager"
+
+// Helper class to extract the credential manager work. Also, caller is in Java, and we need kotlin.
+object PasskeyManager {
```

Comment:
(⛏️ ) You can make this an object and inject the dispatchers to make the class more testable. It is a nitpick, so you can disregard this comment.

Reply:
I am thinking about this. For now I will not do that. Part of the reason is because I wouldn't unit test this class, hence why I have static methods. It heavily uses 3rd party SDK, which has very little value to unit test. I'd be mocking everything anyways. That being said, I will pass in the dispatcher instead of directly doing `Dispatcher.IO`.